src/gwt/src/org/rstudio/studio/client/workbench/MRUList.java

```diff
@@ -128,6 +129,7 @@ private void updateCommands()
       if (hideClearOnEmpty_)
          clearCommand_.setVisible(clearCommand_.isEnabled());
       manageCommands(mruEntries_, mruCmds_);
+      DesktopMenuCallback.commitCommandShortcuts();
```

Comment:
This is required to force Electron to update the menu item labels. Updating the menu item isn't enough and it has to be rebuilt. Better to do it here so it catches all menu items that may have changed visibility, labels, and description.